docs/src/scores.md

```diff
@@ -0,0 +1,174 @@
+# Scores
+
+Generally, the point of running the chain is comparing the properties of some
+existing plan to the properties of the ensemble of plans generated by the chain.
+Thus, we need a way to record information about the plans generated during the
+progression of the chain. This is the point of `Score`s: each score has a
+scoring function that is evaluated on the current districting plan at each step
+of the chain. The resulting values are saved in a dictionary, which is itself
+added to a growing `ChainScoreData` object at each step of the chain.
+The `ChainScoreData` returned by `recom_chain()` and `flip_chain()` is like
+a "history" of the scores at each step of the chain.
+(See [get_scores_at_step](#get_scores_at_step) for information about how to
+retrieve the values of any subset of scores at a particular step in the chain.)
```

Comment:
Just checking - do these links still work in the new documentation?

Reply:
great catch - they didn't! fixed.